mathics/builtin/numbers/algebra.py

```diff
@@ -587,21 +601,24 @@ def coeff_power_internal(self, expr, var_exprs, filt, evaluation, form="expr"):
             var_pats = [Pattern.create(var) for var in var_exprs]
 
         # ###### Auxiliary functions #########
-        def key_powers(lst):
+        def key_powers(lst: list) -> int:
```

Comment:
I think we can return a float, right? So it would be `Union[int, float]`, I think.